docs/configuration/configuration.md

```diff
@@ -1302,6 +1306,66 @@ tls_config:
   [ <tls_config> ]
 ```
 
+### `<eureka_sd_config>`
+
+Eureka SD configurations allow retrieving scrape targets using the
+[Eureka](https://github.com/Netflix/eureka) REST API. Prometheus
+will periodically check the REST endpoint for currently running tasks and
+create a target group for every app.
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+* `__meta_eureka_app_name`: the name of the app (with slashes replaced by dashes)
```

Comment:
Why do we do replace this?

Reply:
We don't need a replace. The explanation is wrong. I'll fix it.